website/events/views.py

```
@@ -31,8 +46,64 @@ def get_serializer_context(self):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        serializer = EventLikeSerializer(data=request.data)
+        if serializer.is_valid():
+            event_id = serializer.validated_data['event_id']
+            event = Event.objects.get(pk=event_id)
+            like, created = Like.objects.get_or_create(user=request.user, event=event)
+            if created:
+                return Response(status=status.HTTP_201_CREATED)
+            else:
+                return Response(status=status.HTTP_200_OK)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)